boards/BMS/Src/Io/Io_LTC6813/Io_LTC6813CellTemperatures.c

```diff
@@ -0,0 +1,190 @@
+#include <string.h>
+#include "Io_SharedSpi.h"
+#include "Io_LTC6813/Io_LTC6813Shared.h"
+
+// clang-format off
+
+// Commands used to read from auxiliary register groups A-C
+#define RDAUXA (0x0C00U)
+#define RDAUXB (0x0E00U)
+#define RDAUXC (0x0D00U)
+
+// Command used to trigger ADC conversions for LTC6813 GPIO inputs
+#define ADAX ((uint16_t)((((MD << 7U) + 0x0060U + CHG) << 8U) | 0x0004U))
+
+#define SIZE_OF_TEMPERATURE_LUT (201U)
+#define BIAS_RESISTOR_OHM       (10000.0f)
+#define REFERENCE_VOLTAGE       (3.0f)
+#define THERM_INDEX_TO_DEGC     (5U)
+
+// clang-format on
+
+enum AuxiliaryRegisterGroup
+{
+    AUX_REGISTER_GROUP_A = 0U,
+    AUX_REGISTER_GROUP_B,
+    AUX_REGISTER_GROUP_C,
+    NUM_OF_AUX_REGISTER_GROUPS
+};
+
+extern struct SharedSpi *ltc6813_spi;
```

Comment:
Should we assert that `ltc6813_spi` != NULL within Io_LTC6813Shared.c to ensure that the pointer has been initialized. I think it makes sense to, curious what your thoughts are.

Reply:
I think we should be okay without the assertion because ltc6813_spi is guaranteed to be initialized in main.c before any of these functions are called in the state machine

Reply:
we are checking to see if the spi handle provided is NULL when we init ltc6813_spi, so there is a layer of safety there (if thats what you were concerned about as well)